ui/apps/platform/src/utils/fileUtils.ts

```diff
@@ -0,0 +1,5 @@
+const filenameSanitizerRegex = new RegExp('(:)|(/)|(\\s)', 'gi');
+
+export function sanitizeFilename(filename: string, replacementChar: string = '_') {
```

Comment:
Super, can you share the existence of this utility function at next team meeting.